scripts/install_environment.sh

```diff
@@ -63,6 +65,24 @@ function install_libvirt() {
         socat \
         tigervnc-server
 
+    if [ $? -ne 0 ]; then
+        echo "Installation with CRB repo failed, trying without CRB..."
+
+        # Try to install without enabling the CRB repository
+        sudo dnf install -y \
+            libvirt \
+            libvirt-devel \
+            libvirt-daemon-kvm \
+            qemu-kvm \
+            libgcrypt \
+            swtpm \
+            swtpm-tools \
+            socat \
+            tigervnc-server
```

Comment:
for maintenance reasons, I think the list of the packages should be defined once

Reply:
Agree